make/Build.gmk

```diff
@@ -84,8 +84,10 @@ $(JEXTRACT_IMAGE_DIR): $(BUILD_MODULES_DIR)
 	    --module-path "$(BUILD_MODULES_DIR)" \
 	    --add-modules org.openjdk.jextract \
 	    --add-options "\"--enable-native-access=org.openjdk.jextract\" \"--enable-preview\""
-	$(CP) src/main/jextract "$(JEXTRACT_IMAGE_DIR)"
-	$(CP) src/main/jextract.bat "$(JEXTRACT_IMAGE_DIR)"
+	$(MKDIR) $(JEXTRACT_IMAGE_DIR)/bin
+	$(CP) src/main/jextract "$(JEXTRACT_IMAGE_DIR)/bin"
+	$(CP) src/main/jextract.bat "$(JEXTRACT_IMAGE_DIR)/bin"
+	$(CHMOD) +x "$(JEXTRACT_IMAGE_DIR)/bin/jextract"
```

Comment:
This line is partly unrelated, but it does update the file permissions of the Linux shell, similarly to what happens on Gradle.